Startup/Setup.js

```diff
@@ -1,759 +1,829 @@
+
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+    askForToy("Shock Collar");
+    askForToyUsage("ShockCollar", domChose);
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be quite painful", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("estimPainHigh", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be somewhat painful", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("eStimPainMedium", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+    sendVirtualAssistantMessage("Tell me what level of shock you consider to be less painful and maybe even pleasant", false)
+    answer = createInput();
+
+    while (true) {
+        if (answer.isDouble()) {
+            setVar("eStimPainLow", answer.getDouble());
+            break;
+        } else {
+            sendVirtualAssistantMessage("This is not a valid number. Please just type a number such as 1, or 2.5");
+            answer.loop();
+        }
+    }
+
+		
+
+	
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+	askForToy("EStim");
+    askForToyUsage("EStim", domChose);
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+
+    setupGags(true);
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+    askForToy("Girl friend");
+	      if(getVar("toyGirlfriend")==true)  
+    		  {sendVirtualAssistantMessage("So tell me the name of your girlfriend", false);
+    			setVar("girlfriendname", createInput().getAnswer());
+    		  }
+      
+      
+      
+      
+	   BASIC_LINGERIE.askForToyAndUsage(domChose, undefined, "LingerieSet");
+     ADVANCED_LINGERIE.askForToyAndUsage(domChose, undefined, "LingerieSet2");
+
+     PARACHUTE_TOY.askForToyAndUsage(domChose);
+     showImage("Images/Spicy/Toys/LingerieSet*.jpg",3);
+  
+    sendVirtualAssistantMessage(random("Okay then...", "Next...", "Let's see...", "Moving on..."));
+	showImage("Images/Spicy/Toys/HotSauce.jpg",3);
+    sendVirtualAssistantMessage("Hot sauce or icy hot? Toothpaste can work too for the time being.", false);
+    showPicture("Images/Spicy/Toys/HotSauce.jpg");
+
+    answer = createInput();
+    const variableName = "hotSauce";
+    while (true) {
+        if (answer.isLike("yes")) {
+            setVar("toy" + variableN
```

Comment:
???

Reply:
it was a few months ago that I did this, but I seem tp recall there was a missing paren (line below) that crashed setup.